content/pt/docs/concepts/instrumentation/code-based.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Baseada em código
+description:
+  Informe-se sobre as etapas essenciais na configuração da instrumentação
+  baseada em código
+weight: 20
+aliases: [manual]
+default_lang_commit: fe0c3f68902aeb6e7584ffc32c6c8964d7a89e6e
+---
+
+## Importar a API e o SDK do OpenTelemetry
+
+Primeiro, você precisará importar o OpenTelemetry para o seu código. Se estiver
+desenvolvendo uma biblioteca ou algum outro componente que se destina a ser
+consumido por um binário executável, você só dependeria da API. Se o seu
+artefato for um processo ou serviço independente, então você dependeria da API e
+do SDK. Para obter mais informações sobre a API e o SDK do OpenTelemetry,
+consulte a [especificação](/docs/specs/otel/).
+
+## Configurar a API do OpenTelemetry
+
+Para criar rastros ou métricas, primeiro você precisa criar um provider tracer
+e/ou meter. Geralmente, recomendamos que o SDK forneça um único provider padrão
+para esses objetos. Em seguida, você obtém uma instância tracer ou meter desse
+provider e atribui a ela um nome e uma versão. O nome que você escolher aqui
+deve identificar exatamente o que está sendo instrumentado - se você está
+desenvolvendo uma biblioteca, por exemplo, você deve nomeá-la com o nome da sua
+biblioteca (por exemplo `com.example.myLibrary`), pois esse nome será usado como
+_namespace_ em todos os eventos de trecho ou métricas que forem produzidos. Além
+disso, é recomendável fornecer uma string de versão (ou seja, `semver:1.0.0`)
+que corresponda à versão atual da sua biblioteca ou serviço.
+
+## Configurar o SDK do OpenTelemetry
+
+Se você está desenvolvendo um processo de serviço, também precisará configurar o
+SDK com as opções adequadas para exportar seus dados de telemetria para um
+backend de análise. Recomendamos que esta configuração seja feita
+programaticamente por meio de um arquivo de configuração ou outro mecanismo.
+Além disso, existem opções de ajuste específicas para cada linguagem de
+programação que você pode utilizar.
+
+## Criar dados de telemetria
+
+Depois de configurar a API e o SDK, você estará livre para criar eventos de
+rastro e métricas usando os objetos tracer e meter obtidos do provider. Utilize
+bibliotecas de instrumentação para suas dependências -- confira o
+[registry](/ecosystem/registry/) ou o repositório da sua linguagem de
```

Comment:
Hum....."registry" é mais um termo que eu não sei se é comumente utilizado em inglês ou se deveria ser traduzido.

E se for para traduzir eu não sei qual termo utilizar. "Registro" não perece muito certo, "arquivo" (de "archive") é meio ambíguo.

@edsoncelio @EzzioMoreira alguma sugestão?

Reply:
https://opentelemetry.io/ecosystem/registry/ é um mecanismo de busca.

Reply:
Ah sim, minha pergunta foi mais no sentido de se devemos ou não traduzir a palavra "registry" 🙂

Reply:
Eu colocaria `OpenTelemetry Registry`